src/test/regress/expected/multi_simple_queries.out

```diff
@@ -664,6 +664,62 @@ DEBUG:  query has a single distribution column value: 1
  41 |         1 | aznavour     |      11814
 (5 rows)
 
+SELECT count(*) FROM articles TABLESAMPLE SYSTEM (0);
+DEBUG:  Router planner cannot handle multi-shard select queries
+ count
+---------------------------------------------------------------------
+     0
+(1 row)
+
+SELECT count(*) FROM articles TABLESAMPLE BERNOULLI (0);
+DEBUG:  Router planner cannot handle multi-shard select queries
+ count
+---------------------------------------------------------------------
+     0
+(1 row)
+
+SELECT count(*) FROM articles TABLESAMPLE SYSTEM (100);
+DEBUG:  Router planner cannot handle multi-shard select queries
+ count
+---------------------------------------------------------------------
+    50
+(1 row)
+
+SELECT count(*) FROM articles TABLESAMPLE BERNOULLI (100);
+DEBUG:  Router planner cannot handle multi-shard select queries
+ count
+---------------------------------------------------------------------
+    50
+(1 row)
+
+SELECT count(*) FROM (SELECT random() FROM articles TABLESAMPLE SYSTEM (0)) a;
+DEBUG:  Router planner cannot handle multi-shard select queries
+ count
+---------------------------------------------------------------------
+     0
+(1 row)
+
+SELECT count(*) FROM (SELECT random() FROM articles TABLESAMPLE BERNOULLI (0)) a;
+DEBUG:  Router planner cannot handle multi-shard select queries
+ count
+---------------------------------------------------------------------
+     0
+(1 row)
+
+SELECT count(*) FROM (SELECT random() FROM articles TABLESAMPLE SYSTEM (100)) a;
+DEBUG:  Router planner cannot handle multi-shard select queries
+ count
+---------------------------------------------------------------------
+    50
```

Comment:
Are there only 50 rows in the `articles` table?

Reply:
> Are there only 50 rows in the articles table

yes, there are 50 numbered inserts above.